tests_config/run_spec.js

```diff
@@ -305,6 +344,20 @@ function runTest({
       expect(output).toEqual(expected);
     });
   }
+  if (shouldCompareBytecode(filename, formatOptions)) {
+    test(`[${parser}] compare Bytecode`, () => {
+      // We require the compiler here as it makes all tests slow when added at
+      // the top of the file.
+      // TODO investigate this warning
+      //   - A worker process has failed to exit gracefully and has been force
+      //     exited. This is likely caused by tests leaking due to improper
+      //     teardown. Try running with --detectOpenHandles to find leaks.
+      const compileContract = require('./utils/compile-contract');
```

Comment:
I'd appreciate if someone could investigate how to include the compiler, without incurring in greater testing times.

Reply:
There is a way to improve the time: download a native compiler and use that instead. It's significantly faster. But this means:

1. Downloading the correct compiler according to the OS (and falling back to the WASM one if the OS is not supported)
2. Avoiding a re-download if it was previously download
3. (Optional) Cache it in the CI

I can do this since we do something similar (even more involved) in Hardhat.

That being said, it takes 1m to do this, so I don't think it's worth it? I'd rather re-visit this in the future if it's a problem, but for now I would say that we just run `npm test` during development and use `npm run test:all` before pushing (or just checking the result of the CI).